apps/extension/webpack.config.ts

```diff
@@ -0,0 +1,153 @@
+import * as dotenv from 'dotenv';
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import url from 'url';
+// eslint-disable-next-line import/no-named-as-default
+import webpack from 'webpack';
+
+// Loads default vars from `.env.testnet` & `.env.mainnet` file in this directory.
+// Reference package.json build script. TODO: add `.env.mainnet` when ready.
+const envPath = process.env['NODE_ENV'] === 'mainnet' ? '.env.testnet' : '.env.testnet';
+dotenv.config({ path: envPath });
```

Comment:
Allows us to pass in different envs per chain (quite relevant when we go to mainnet)